lom/int_test/copy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I skip reviewing rest of the code in this folder.
Let us focus on getting your code changes into LoMProd, so we can have a working LoM service by this Friday.
So let us not increase the  scope.

Next PR: integrate tests into LoM Service as available built-in